app/uk/gov/hmrc/minorentityidentification/repositories/JourneyDataRepository.scala

```diff
@@ -37,7 +37,8 @@ class JourneyDataRepository @Inject()(mongoComponent: MongoComponent,
     collectionName = "minor-entity-identification",
     mongoComponent = mongoComponent,
     domainFormat = implicitly[Format[JsObject]],
-    indexes = Seq(timeToLiveIndex(appConfig.timeToLiveSeconds))
+    indexes = Seq(timeToLiveIndex(appConfig.timeToLiveSeconds)),
+    replaceIndexes = true
```

Comment:
Is this necessary? ... do we need to replace the index once is there? ... you did maybe for test purpose!

Reply:
I think I would  use the default value  86400 maybe write it with a nice name and that's it.
In general is not a problem rebuild the index if the collection is small but I think is not strictly required in this case.
Should we use  86400? I think we should ;-)